boa_tester/src/main.rs

```diff
@@ -390,16 +406,22 @@ enum TestOutcomeResult {
     Panic,
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
```

Comment:
Should also derive `Copy`. We can remove the `clone` calls that way.

Reply:
Oops, Raz already made a comment for this 😅

Reply:
You did it first ;)

Thanks for the tip!